ui/i18n/appointments/locale_en.json

```diff
@@ -229,5 +229,8 @@
   "CHANGE_DATE_TO_LABEL": "Change ${moment(originalAppointmentDate).format('Do MMM')} to",
   "PATIENT_DOUBLE_BOOKING_CONFLICT_MESSAGE": "Current {currentAppointmentService} request conflicts with {existingAppointmentService} appointment on and",
   "NO_SERVICE_CONFLICTS_DEFAULT_TEXT": "The {label} service you had selected for the appointment(s) is not available during below listed dates",
-  "PROVIDER_ERROR_MESSAGE": "Please select only a maximum of {maxAppointmentProviders} provider(s)"
+  "PROVIDER_ERROR_MESSAGE": "Please select only a maximum of {maxAppointmentProviders} provider(s)",
+  "PROVIDERS": "Providers",
```

Comment:
These keynames are the same as the key names in the file above. What is the difference in both. Is one "headings"? I feel the Key name should be more specific where this is being used.